phys/module_physics_init.F

```diff
@@ -4380,7 +4380,7 @@ SUBROUTINE mp_init(RAINNC,SNOWNC,GRAUPELNC,config_flags,restart,warm_rain,
 !------------------------------------------------------------------
    USE module_mp_wsm3
    USE module_mp_wsm5
-   USE module_mp_wsm6
+   USE mp_wsm6
```

Comment:
I think the corresponding depend in depend.common needs to change to point to this module now

Reply:
@islas This should be fixed.